components/proxy/src/main/java/com/hotels/styx/startup/StyxPipelineFactory.java

```diff
@@ -90,11 +91,12 @@ private static HttpHandler styxHttpPipeline(StyxConfig config, HttpHandler inter
             builder.add(new HttpMessageLoggingInterceptor(longFormatEnabled));
         }
 
-        builder.add(new ConfigurationContextResolverInterceptor(EMPTY_CONFIGURATION_CONTEXT_RESOLVER));
-        builder.add(new UnexpectedRequestContentLengthRemover());
-        builder.add(new ViaHeaderAppendingInterceptor());
-        builder.add(new HopByHopHeadersRemovingInterceptor());
-        builder.add(new RequestEnrichingInterceptor(config.styxHeaderConfig()));
+        builder.add(new TcpTunnelRequestRejector()).
```

Comment:
it seems that you use different convention with the dot at the end instead of at the front